regress/tests/test-029.c

```diff
@@ -0,0 +1,101 @@
+#pragma strict_types
```

Comment:
I was able to test the strlen and break_string changes easily enough, but I have no idea how to test the change to write_socket.